test/ranges.jl

```diff
@@ -710,13 +710,13 @@ function test_linspace_identity(r::Range{T}, mr) where T
     @test -collect(r) == collect(mr)
     @test isa(-r, typeof(r))
 
-    @test 1 + r + (-1) == r
-    @test 1 + collect(r) == collect(1 + r) == collect(r + 1)
-    @test isa(1 + r + (-1), typeof(r))
-    @test 1 - r - 1 == mr
-    @test 1 - collect(r) == collect(1 - r) == collect(1 + mr)
-    @test collect(r) - 1 == collect(r - 1) == -collect(mr + 1)
-    @test isa(1 - r - 1, typeof(r))
+    @test 1 .+ r .+ (-1) == r
+    @test 1 .+ collect(r) == collect(1 .+ r) == collect(r .+ 1)
+    @test_broken isa(1 .+ r .+ (-1), typeof(r))
```

Comment:
Oh god, I'm in some kind of hell where (a) I can't use stacktraces to figure out why my sysimg won't build and (b) we still have the v0.6 deprecations and some of my new range broadcasts are somehow being taken over by the old `.+` operator or something...